app/src/main/res/drawable/ic_national_speed_limit.xml

```diff
@@ -0,0 +1,17 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="342.05dp"
+    android:height="342.05dp"
```

Comment:
Looks a bit big to me. Best look at the size of the other signs and adapt it.

Anyway, I really wonder why the path data is so big for that sign, isn't it just a circle with a stroke through it?